related-tooling.md

```diff
@@ -35,6 +35,14 @@ It is currently working towards [Phase 1 of the Kubernetes Operator Capability M
 
 See [latest slide deck](http://people.redhat.com/mdepaulo/presentations/Introduction%20to%20pulp-operator.pdf) for more info.
 
+## pulp-insta-demo.sh
+
+[A script](https://raw.githubusercontent.com/pulp/pulp-operator/master/insta-demo/pulp-insta-demo.sh) to install Pulp 3 on Linux systems with as many plugins as possible and an uninstaller.
+
+Works by installing [K3s (lightweight kubernetes)](https://k3s.io/), and then deploying pulp-operator on top of it.
+
+Is not considered production because pulp-operator is not production yet, and it hides every config options. Only Suitable as a quick wat to evaluate Pulp for the time being.
```

Comment:
```suggestion
Is not considered production ready because pulp-operator is not yet, and it hides every config options. Only suitable as a quick way to evaluate Pulp for the time being.
```

Reply:
Incorporated in my latest commit.